adapters/lora/src/main/java/org/eclipse/hono/adapter/lora/providers/JsonBasedLoraProvider.java

```diff
@@ -177,7 +177,7 @@ protected UplinkLoraMessage createUplinkMessage(final HttpServerRequest request,
 
         Objects.requireNonNull(requestBody);
 
-        final String devEui = getDevEui(requestBody);
+        final String devEui = LoraUtils.convertToHexString(getDevEui(requestBody));
```

Comment:
are you sure that you want to encode the bytes as HEX instead of Base64?
FMPOV the latter is much easier to process and is also more compact ...

Reply:
yes, this is a non-breaking change for the business application since they were using HEX encoding before.
Secondly, all users are talking about the HEX encoded devEUI (and this is much more readable compared to Base64 encoded).

Reply:
ok then, you're the expert :+1: